common/src/main/scala/models/Notification.scala

```diff
@@ -134,3 +136,25 @@ object ElectionNotification {
   import JsonUtils._
   implicit val jf = Json.format[ElectionNotification]
 }
+
+case class LiveEventNotification(
+  id: UUID,
+  `type`: NotificationType = LiveEventAlert,
+  sender: String,
+  title: String,
+  message: String,
+  expandedMessage: Option[String],
+  shortMessage: Option[String],
+  importance: Importance,
+  link1: Link, // where does the title for this come from?
+  link2: Link, // where does the title for this come from?
```

Comment:
I'm a little confused by these two comments. Are both links where the title comes from ?